tracer/test/test-applications/debugger/dependency-libs/Samples.Probes.External/SecurityTransparentTest.cs

```diff
@@ -0,0 +1,17 @@
+
+using System;
+using System.Diagnostics;
+using System.Security;
+
+[assembly: SecurityTransparent]
+//[assembly: SecurityRules(SecurityRuleSet.Level2)]
```

Comment:
```suggestion
```
Please remove it if we don't need it

Reply:
Done in bf6df15958c13137a9c5bbf36c849990bc3d909e.